src/highdicom/sr/utils.py

```diff
@@ -174,3 +175,35 @@ def get_coded_value(item: Dataset) -> CodedConcept:
         meaning=value.CodeMeaning,
         scheme_version=value.get('CodingSchemeVersion', None)
     )
+
+
+def get_coded_modality(item: Dataset) -> str:
+    """
+    Gets the coded value of the modality from the dataset's SOPClassUID. The
+    SOPClassUIDs are defined here:
+    http://dicom.nema.org/dicom/2013/output/chtml/part04/sect_B.5.html
+    and the coded values are described here:
+    http://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_29.html
```

Comment:
Could you please format this part of the docstring according to reStructuredText and the NumPy docstring convention?

Reply:
Done. Thanks.